setup/build_environment.py

```diff
@@ -88,14 +88,16 @@ def readvar(name):
 from PyQt5.QtCore import PYQT_CONFIGURATION
 pyqt['sip_flags'] = PYQT_CONFIGURATION['sip_flags']
 def get_sip_dir():
-    q = os.environ.get('SIP_DIR', sys.prefix if iswindows else os.path.join(sys.prefix, 'share', 'sip'))
```

Comment:
this seems not right - are you sure about this change?

Reply:
You are correct, that change was necessary to get the code working in my debug environment. It does not belong in the PR.